nomad/structs/structs.go

```diff
@@ -863,7 +866,6 @@ type EvalDequeueRequest struct {
 type EvalListRequest struct {
 	FilterJobID      string
 	FilterEvalStatus string
-	OrderAscending   bool
```

Comment:
Removing this field broke a few tests in `eval_endpoint_test.go` and `deployment_endpoint_test.go`

Reply:
Thanks! I though a `make test` would be enough to check if the code compiled 😅